packages/atclient/include/atclient/socket_shared.h

```diff
@@ -0,0 +1,57 @@
+// IWYU pragma: private, include "atclient/net_socket.h"
+// IWYU pragma: friend "net_socket_mbedtls.*"
```

Comment:
TODO replace instances of net_socket with the updated file names

Reply:
Addressed in [40f6221](https://github.com/atsign-foundation/at_c/pull/475/commits/40f6221f743d7536dd9dd3fa102657fb0bc8230f)